test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -244,6 +367,109 @@ def func(x, b=torch.tensor(1.0)):
             self, func, (tensor_x,), b=torch.tensor(5.0)
         )
 
+    # beartype.roar.BeartypeCallHintParamViolation:
+    # @beartyped onnxscript.function_libs.torch_aten.graph_building.TorchScriptGraph.add_input()
+    # parameter input_value=8.0 violates type hint typing.Union[torch.Tensor, NoneType],
+    # as float 8.0 not <class "builtins.NoneType"> or <protocol "torch.Tensor">.
+    # @unittest.expectedFailure
+    @pytorch_test_common.skip_fx_exporters(
+        {
+            DynamoExporter: (
+                "beartype.roar.BeartypeCallHintReturnViolation: @beartyped "
+                "torch.onnx._internal.exporter.ExportOutput.adapt_torch_inputs_to_onnx() "
+                "return (tensor([[[ 1.5410, -0.2934]]]), 8.0) violates type hint "
```

Comment:
We can, it can happen in follow ups though. I'm unsure yet if we'd need adjustments on fx_to_onnx conversion pass, so don't want to extend the scope of this PR.